### PR TITLE
Week 7 schedule updates

### PR DESCRIPTION
**Pull request by oliverjam**
- Add Monday afternoon workshop
- Add spikes
- Add project

### CODE REVIEW

**Comment by sofer** [comment]
Minors:
- stretch goals on linked Middleware workshop: refactoring before fancier error logging, perhaps?
- I didn't see anything on the mocking learning outcome (yet),  or did I miss it?

**Comment by oliverjam** [comment]
Yeah I guess swapping the stretch goals would make sense.

The original plan was for a morning challenge on mocking, but I think it's new enough, important enough and confusing enough that it deserves the 90 minute workshop slot on Tuesday afternoon. We can find something else for the MC

File: week-7/schedule.md [hunk 1]
```diff
@@ -2,41 +2,44 @@
...
+| ------- | --------------------- | ----------------- |
+| 09:45   | Check-in              |                   |
+| 10:00   | Spike presentations   |                   |
+| 11:00   | [REST APIs][rest-api] | REST, JSON API    |
+| _13:00_ | _Lunch_               |                   |
+| 14:00   | Test mocking          | Mocking           |
+| 15:30   | Employment prep       |                   |
+| 16:30   | Tech for Better       |                   |
+
+[rest-api][https://github.com/oliverjam/learn-rest-apis]
```
Comment (Albadylic) [inline week-7/schedule.md:31]: Broken link here

Comment (oliverjam) [reply week-7/schedule.md:31]: Oops 😅 I'll push that fix to Master

**Pull request merged**